Yoast/yoast-components
# Updated Node-sass, Grunt-sass  & Jest-cli to clean up subdependencies.

## Summary

This PR can be summarized in the following changelog entry:

* N/A. Updated Node-sass, Grunt-sass  & Jest-cli to clean up subdependencies. 

## Relevant technical choices:

* The original issue was to update Cryptiles to 4.1.2, but instead it has been removed as a dependency. Cryptiles was a dependency of hawk. Which was a dependency or request. Request was used by Node-sass, Grunt-sass  & Jest-cli, and this resulted in 3 different versions of request in the yarn.lock file. I have updated all 3 to their latest versions which now all use the latest request(2.88) version. This has two advantages: 
-only one request version is used instead of several.
-request does not use hawk (and therefore cryptiles) anymore since version 2.87 so instead of updating these dependencies we simply have removed them. 
* Updates Merge to 1.2.1
* Updates Randomatic to 3.0.0

* After updating an error was thrown in the Travis, this was solved by adding an testurl to the jest configuration as noted here: https://github.com/facebook/jest/issues/6766


## Test instructions

This PR can be tested by following these steps:

* Checkout this branch.
* Just to be sure, delete the `node_modules` and `dist` folder. 
* Rebuild all the files by running yarn and grunt. 
* All tests should pass, nothing should break. There should be no changes. 

## UI changes

Fixes #837
Fixes #838 
Fixes #839
